python/hail/methods/impex.py

```diff
@@ -8,6 +8,29 @@
 from hail.expr.expressions import analyze, expr_any
 from hail.genetics.reference_genome import reference_genome_type
 from hail.methods.misc import require_biallelic, require_row_key_variant
+import hail as hl
+
+
+def locus_interval_expr(contig, start, end, includes_start, includes_end, reference_genome, skip_invalid_intervals):
+    if reference_genome:
+        if skip_invalid_intervals:
+            is_valid_locus_interval = (hl.is_valid_contig(reference_genome, contig) &
+                                       (hl.is_valid_locus(reference_genome, contig, start) | (~includes_start & (start == 0))) &
+                                       (hl.is_valid_locus(reference_genome, contig, end) | (~includes_end & hl.is_valid_locus(reference_genome, contig, end - 1))))
```

Comment:
line too long. If you open a paren after is_valid_locus_interval, then the other stuff can go on a new line.